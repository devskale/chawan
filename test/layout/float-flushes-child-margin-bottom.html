<div style="float: left">
<div style="margin-bottom: 1em">
testing testing
</div>
</div>
<div>
test<br>
test<br>
test<br>
test<br>
