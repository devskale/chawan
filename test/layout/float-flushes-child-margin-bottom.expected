testing testingtest
               test
test
test
